addons/mrp_account/report/report_mrp_templates.xml

```diff
@@ -9,7 +9,9 @@
                     <div class="row mt8">
                         <div class="col-lg-12">
                             <h2>
-                                <span>WIP Report for <span t-field="docs.account_id.name">Acme Corp.</span></span>
+                                <span> WIP Report for
```

Comment:
I did not notice we add a space before `WIP` in the previous PR, is it really needed?

Reply:
@kcv-odoo ping. 🙂

Reply:
Not I guess :innocent: 
Should be fine now! :100: